test/jdk/java/nio/channels/DatagramChannel/Disconnect.java

```diff
@@ -52,7 +52,7 @@ public static void main(String[] args) throws IOException {
         if (IPSupport.hasIPv4()) {
             // test with IPv4 only
             try (DatagramChannel dc = DatagramChannel.open(StandardProtocolFamily.INET)) {
-                InetAddress lo4 = InetAddress.ofLiteral("127.0.0.1");
+                InetAddress lo4 = Inet4Address.getByName("127.0.0.1");
```

Comment:
`Inet4Address` doesn't override `getByName()` we should use `InetAddress` instead.

```suggestion
                InetAddress lo4 = InetAddress.getByName("127.0.0.1");
```